dist/index.html

```diff
@@ -15,28 +15,23 @@ <h1>Leaderboard</h1>
       <section class="scores-section">
```

Comment:
- Kindly enable the correct  [linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) in your Pull Request.